npm ERR! Invalid tag name ">=null": Tags may not have any characters that encodeURIComponent encodes.

npm install fails with the below error due to invalid dependency tag name.

```
$ npm install --save datatables.net-staterestore-bs4
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name ">=null": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ctrought/.npm/_logs/2022-01-03T19_31_06_186Z-debug-0.log
```

https://github.com/DataTables/Dist-DataTables-StateRestore-Bootstrap4/blob/aa55df98ae54f0f0c7c51a7b382aa8a3006d734d/package.json#L27

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
         "sort"
     ],
     "dependencies": {
-        "datatables.net-staterestore": ">=null",
+        "datatables.net-staterestore": ">=1.0.1",
         "datatables.net-bs4": ">=1.11.3",
         "jquery": ">=1.7"
     },

